content/en/blog/2024/otel-governance/index.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Behind the scenes of the OpenTelemetry Governance Committee
+date: 2024-08-15
+author: '[Juraci Paixão Kröhling](https://github.com/jpkrohling) (Grafana)'
+issue: 5033
+---
+
+![Chart showing what technologies are being used](header.png)
```

Comment:
Chart? Copy/paste issue? In any case, feel free to drop the header, as it has the Grafana logo in it, which makes it weird for the OTel publication.